packages/jsts/src/rules/S2871/unit.test.ts

```diff
@@ -606,6 +606,8 @@ ruleTester.run(
         code: 'const array = ["foo", "bar"]; const sortedArray = array.toSorted();',
         errors: [
           {
+            message:
+              'Provide a compare function that depends on String.localeCompare, to reliably sort elements alphabetically.',
```

Comment:
Might need to apply this suggestion if you applied the one I suggested above.

```suggestion
              'Provide a compare function that depends on "String.localeCompare", to reliably sort elements alphabetically.',
```